lerna.json

```diff
@@ -11,7 +11,11 @@
     }
   },
   "loglevel": "success",
-  "packages": ["packages/*"],
+  "packages": [
+    "packages/*",
+    "themes/gatsby-theme-blog",
```

Comment:
Is there some other heuristic we could use?

If we don’t want them published — what about `private: true` or something?

(also if this has bee discussed and I’ve missed it — ignore!)

Reply:
The `themes` folder contains two themes packages as well. This is temporary till they're moved out to `/starters` so I didn't bother.